packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Checkbox } from '../Checkbox';
+import { FieldErrorMessage, FieldDescription } from '../Field';
+import { Flex } from '../Flex';
+import { CheckboxFieldProps } from '../types/checkboxField';
+import { ComponentClassNames } from '../shared';
+
+export const CheckboxField: React.FC<CheckboxFieldProps> = ({
+  alignContent,
+  alignItems,
+  checked,
+  children,
+  className,
+  defaultChecked,
+  descriptiveText,
+  direction = 'column',
```

Comment:
This should be set in CSS, not as a render style prop.

Reply:
Well, I think This is how we structure other fields? Users can pass the flex props directly instead of changing CSS variables